Foundation/NSConcreteValue.swift

```diff
@@ -91,17 +91,17 @@ internal class NSConcreteValue : NSValue {
 
         self._typeInfo = typeInfo!
 
-        self._storage = UnsafeMutableRawPointer.allocate(bytes: self._typeInfo.size, alignedTo: 1)
-        self._storage.copyBytes(from: value, count: self._typeInfo.size)
+        self._storage = UnsafeMutableRawPointer.allocate(byteCount: self._typeInfo.size, alignment: 1)
+        self._storage.copyMemory(from: value, byteCount: self._typeInfo.size)
```

Comment:
This one doesn't read as an improvement to me - the previous API seemed quite clear as to what it would do. Maybe you are fixing some overloading of the meaning of `count` though? I'm not sure.

Reply:
this was discussed heavily during the review of SE-0184. The core team felt strongly that it should be `copyMemory` and `byteCount`.